### PR TITLE
Punishes the playerbase

### PR DESCRIPTION
**Pull request by Partheo**
image

:cl:  Partheo
tweak: flour sacks will now break if you fill them with water
tweak: scooping reagents from the floor now takes time
/:cl:

### CODE REVIEW

**Comment by Nickvr628** [comment]
No, this is bad

**Comment by Marmio64** [comment]
> No, this is bad

do you hate bowls?

**Comment by Nickvr628** [comment]
as a chef player I feel personally attacked that I cannot use my flour sacks to mix dough

**Comment by maxion12345** [comment]
It's a quality of life thing your removing I don't want to have to measure flour for bread it gets annoying :(

**Comment by Ratvar** [comment]
Nothing in this pr makes you measure flour, thankfully

**Comment by boodaliboo** [comment]
seems like a unneeded difficulty thing

File: code/modules/food_and_drinks/food/condiment.dm [hunk 1]
```diff
@@ -234,6 +234,28 @@
 	list_reagents = list(/datum/reagent/consumable/flour = 30)
 	possible_states = list()
 
+/obj/item/reagent_containers/food/condiment/flour/on_reagent_change(changetype) //born of intense hatred
+	if(!reagents.has_reagent(/datum/reagent/water, 10))
+		return ..()
+	else
```
Comment (JamieD1) [inline code/modules/food_and_drinks/food/condiment.dm]: Not needed, remove the else and tab it all out. Returns dont need else statements.

File: code/modules/food_and_drinks/food/condiment.dm [hunk 2]
```diff
@@ -234,6 +234,28 @@
...
+		var/target
+		var/L
+		for(var/obj/structure/table/S in src.loc)
+			target = S
+			L = S.loc
+		if(!target)
+			var/turf/T = get_turf(src)
+			target = T
+			L = target
+
```
Comment (JamieD1) [inline code/modules/food_and_drinks/food/condiment.dm]: This is horrible, check discord

**Review by JamieD1 — CHANGES_REQUESTED** [review]

File: code/modules/food_and_drinks/food/condiment.dm (hunk 2 above)
Comment (Partheo) [reply code/modules/food_and_drinks/food/condiment.dm]: Any optimizations I've tried to make have broken functionality in some way; this is still the simplest way I can find that makes everything work

**Review by JamieD1 — APPROVED** [review]

**Comment by TheBimmer231** [comment]
This is the worst PR ever, who thought of this, I can't believe you did this Partheo

**Comment by TheBimmer231** [comment]
the chef union will hear about this

**Pull request merged**